src/lib/isIPRange.js

```diff
@@ -0,0 +1,22 @@
+import assertString from './util/assertString';
+
+const ipv4RangeMaybe = /^(\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})\/(\d{1,2})$/;
+
+export default function isIPRange(str, version = '') {
+  assertString(str);
+  version = String(version);
+  if (!version) {
+    return isIPRange(str, 4) || isIPRange(str, 6);
+  } else if (version === '4') {
+    if (!ipv4RangeMaybe.test(str)) {
+      return false;
+    }
+    const parts = str.split('.').sort((a, b) => a - b);
```

Comment:
Can you `isIP` here instead of duplicating the validator? Split by `/` first, validate the first part is an IP address using `isIP(firstPart, 4)`, then validate the range here.